### PR TITLE
fix: Notification list scroll behaviour

### PR DESCRIPTION
**Pull request by olerichter00**
## Description

This PR fixes the scrolling behavior of the Activity list in the split view, which was broken after updating the design of the activity list (https://github.com/artsy/force/issues/13392).

Basically, it wasn't possible to scroll the list independently.

### CODE REVIEW

File: src/Components/Notifications/NotificationsWrapper.tsx [hunk 1]
```diff
@@ -44,9 +44,13 @@ export const NotificationsWrapper: React.FC<NotificationsWrapperProps> = ({
             <Separator />
           </Sticky>
 
-          <Flex overflow="scroll" flexDirection="column">
+          <Box
+            overflow="scroll"
+            maxHeight={`calc(100vh - ${DROPDOWN_CONTENT_HEIGHT}px)`}
+            pb={2}
```
Comment (olerichter00) [inline src/Components/Notifications/NotificationsWrapper.tsx:50]: Adding this to have some bottom padding below the "Show more" button.

**Review by tam-kis — APPROVED** [review]

**Review by MounirDhahri — APPROVED** [review]
oh! missed this before

**Comment by MounirDhahri** [comment]
To avoid the same regression, do you mind if you add a comment about it to make sure we don't break this again in the future

**Comment by olerichter00** [comment]
> To avoid the same regression, do you mind if you add a comment about it to make sure we don't break this again in the future



Good idea! I've added a comment 👍

**Review by MounirDhahri — APPROVED** [review]
Thanks!

**Pull request merged**